### PR TITLE
Tactical Gear + small changes

### PR DESCRIPTION
**Pull request by Bawhoppen**
Adds actual space-proof armor to go with the long neglected SWAT helm.

New combat knife using nienhaus's sprite.

New assault belt which is a smaller version of the military belt.

Slightly changed what some belts could hold.

Can order the armor+helm, a combat knife, and assault belt from cargo for 60 points (will change price if too cheap)

Made spacesuits crate way cheaper because there was no reason for it to be 80 cargo points for one suit.

:cl:
add: Nanotrasen has opened the availability of tactical SWAT gear to station's cargo department
add: Due to a new trade agreement with Chinese space suit manufacturers, Nanotrasen can now obtain basic space suits much cheaper; This has been reflected in the cargo prices
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
:+1: 
Really really powerful armor values

**Comment by KorPhaeron** [comment]
>cargo can order admin only armour

for what purpose

**Comment by Zilenan91** [comment]
@Bawhoppen Can you make the knife a traitor item

**Comment by Bawhoppen** [comment]
@KorPhaeron what do you mean admin only armor
theres already SWAT helm on derelict, and now they have a real suit to accompany them
@Zilenan91 they already have edaggers, eswords, and other sharp stuff though but maybe I guess

**Comment by The-Albinobigfoot** [comment]
>Chinese space suit manufacturers
Non-canon.

**Comment by Bawhoppen** [comment]
It's canon now bitches 
People are still ethnically chinese in the future

**Comment by The-Albinobigfoot** [comment]
Wait, what can military belts even hold?

**Comment by Bawhoppen** [comment]
Like everything that is a certain size limit I believe

**Comment by Iamgoofball** [comment]
can we get instant knife kills from the back

and does it let us throw the person's knife at nearby people?

**Comment by Bawhoppen** [comment]
I don't know how to code the first one
and its just a slightly higher damage normal knife so maybe?

**Comment by Iamgoofball** [comment]
does it play lines upon successful knife kills

**Comment by Bawhoppen** [comment]
i wish

**Comment by Stenthus** [comment]
this is pretty rad

with this cargo should be at least a little bit more competitive with science in terms of supplying combat gear (which I believe they really should be the best at, not counting mechs which are rare)

I guess it sorta raises the question of how much lethality SHOULD cargo be able to order?

**Comment by RemieRichards** [comment]
@Bawhoppen I redid your changelog, please look at https://github.com/tgstation/-tg-station/wiki/Changelog-Guide

**Comment by Bawhoppen** [comment]
Thanks, I asked phil and he just said just look at someone else's and I guess I didn't copy it well enough

**Comment by Bawhoppen** [comment]
wait do I still need to put my name there then?

**Comment by RemieRichards** [comment]
@Bawhoppen if you don't place a name it takes your github name.

**Comment by Bawhoppen** [comment]
ah ok missed that part glancing at that page
will make sure to check next time

**Comment by RemieRichards** [comment]
@Bawhoppen "If you omit your name, your Github name will be used." :P 
no worries, just keep changelog syntax in mind for the future.

**Comment by Steelpoint** [comment]
I see we've found the new standard issue Cargo Tech uniform.

**Comment by Lumipharon** [comment]
Science has very little in terms of defensive shit they can make (mechs and shield basically), cargo can produce actual armour.

**Comment by AlexanderUlanH** [comment]
>60 points for two spaceproof armor sets with very good protections and complete coverage, a pair of the most robust non-traitor melee weapon, and two extremely robust belts

As someone who often plays cargo tech, :+1:, but it doesn't sound balanced.
I feel like this would be a good candidate for a multitool-hacked console, and for costing just a tad more.

**Comment by KorPhaeron** [comment]
Should probably at least have slowdown 1

**Comment by Bawhoppen** [comment]
it is as slow as a normal space suir,
and I am willing to increase the price

**Comment by Zilenan91** [comment]
It's probably fine as-is since it's just armor and what I think is a 15 brute weapon that come in the form of pickaxes a room away already.

**Comment by Bawhoppen** [comment]
its 20 brute knife, I could decrease it a little I guess

**Comment by Zilenan91** [comment]
Jeez yeah put that down to 15

**Comment by Bawhoppen** [comment]
well that would be the same as a cleaver then, switchblade is 20 and I wanted it to be similar to that

**Comment by KorPhaeron** [comment]
That makes sense keep it at 20

**Comment by Zilenan91** [comment]
But switchblades are antag only items, a role where you're required to kill lots of people, I don't think giving non-antags easy access to 20 brute weapons is a good thing.

**Comment by Bawhoppen** [comment]
I'm going to opt to keep it at 20 for now considering it is relatively costly for cargo and fireaxes are 24 brute and are available roundstart

If its too unbalanced it could be nerfed

**Comment by duncathan** [comment]
There are only two fire axes

**Comment by KorPhaeron** [comment]
@duncathan 

There are unlimited spears (18 damage) and large numbers of stunprods (1 hit and you're dead) out there though.

File: code/game/objects/items/weapons/kitchen.dm [hunk 1]
```diff
@@ -86,6 +88,17 @@
 	attack_verb = list("cleaved", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	w_class = 3
 
+/obj/item/weapon/kitchen/knife/combat
+	name = "combat knife"
+	icon_state = "buckknife"
+	item_state = "knife"
+	desc = "A military combat utility survival knife.."
```
Comment (duncathan) [inline code/game/objects/items/weapons/kitchen.dm]: You have a trailing period

File: code/game/objects/items/weapons/kitchen.dm [hunk 2]
```diff
@@ -86,6 +88,17 @@
...
 
+/obj/item/weapon/kitchen/knife/combat
+	name = "combat knife"
+	icon_state = "buckknife"
+	item_state = "knife"
+	desc = "A military combat utility survival knife.."
+	force = 20
+	throwforce = 20
+	origin_tech = "materials=2;combat=4"
+	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "cut")
```
Comment (duncathan) [inline code/game/objects/items/weapons/kitchen.dm]: Is this needed? Don't these get inherited?

Comment (Bawhoppen) [inline code/game/objects/items/weapons/kitchen.dm]: I didn't want diced to be in the list for the combat knife

**Comment by Stenthus** [comment]
I think it's a bit of a stretch to call a locked 60 point cargo crate something that's "easily accessible"

like sure if you're a cargo tech it's pretty easy, but the amount of jobs with ordering access is pretty slim

cargo can also make/order the following which are substantially more lethal:
tasers/lasers/disablers
chainsaws (and have the gear to surgically attach them)
stun batons
shotguns (and the basic ammo variations)
...

the biggest increase in power here is the increase in ability to provide quality space suits

before cargo techs had to order those really crappy suits, now they can just get these AND the added benefit of armor

**Comment by Bawhoppen** [comment]
>controversial

>actual controversial ones don't get the tag

**Comment by Bawhoppen** [comment]
assault belt now has a fancy schmancy recolored sprite

**Comment by Bawhoppen** [comment]
well this get merged if I remove combat knifes from the crate

**Comment by Bawhoppen** [comment]
ayy.lmao

**Pull request merged**